title: [Resolvido] 'python' não é reconhecido como um comando interno
date: 2018-09-03
author: Gustavo Furtado de Oliveira Alves
category: Util
tags: python, linha de comando, variável de ambiente
image: /images/logo-python.png


